ext/src/ruby_api/func.rs

```diff
@@ -40,8 +43,26 @@ unsafe impl Sync for ShareableProc {}
 unsafe impl Send for Func {}
 
 impl Func {
+    /// @yard
+    /// @def new(store, type, callable, &block)
+    /// @param store [Store]
+    /// @param type [FuncType]
+    /// @param block [Block] The funcs's implementation
+    ///
+    /// @yield [caller, *args] The function's body
+    /// @yieldparam caller [Caller] Caller which can be used to interact with the {Store}.
+    /// @yieldparam *args [Object] Splat of Ruby objects matching the {FuncType}’s params arity.
+    /// @yieldreturn [nil, Object, Array<Object>] The return type depends on {FuncType}’s results arity:
+    ///   * 0 => +nil+
+    ///   * 1 => +Object+
+    ///   * > 1 => +Array<Object>+
+    ///
+    /// @return [Func]
+    ///
+    /// @example
+    ///   store = Wasmtime::Store.new(Wasmtime::Engine.new)
```

Comment:
Is this an example of `Func` usage? 

Seems like a copy-paste error from `Store`.

Reply:
Yes, thanks for catching this. Fixed now:

https://github.com/bytecodealliance/wasmtime-rb/blob/547488dd7e6a8cd218e1714377e490e07cc20f62/ext/src/ruby_api/func.rs#L62-L67